### PR TITLE
ignored c9 files, quoted payload

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

**Comment by kzidane** [comment]
Should remove ` -fsanitize=signed-integer-overflow` from Speller's `Makefile` before merging and deploying.

File: check50-wrapper [hunk 1]
```diff
@@ -15,7 +15,7 @@ tag_sha=$(git commit-tree HEAD^{tree} -m "$tag_name")
 git push origin $tag_sha:refs/tags/$tag_name
 
 # get style50 results
-style50_results=$(unset CHECK50_TOKEN CHECK50_ID; style50 --ignore "*.git*" --output=score .)
+style50_results=$(unset CHECK50_TOKEN CHECK50_ID; style50 --ignore "*.git*" --ignore "*~c9*" --output=score .)
```
Comment (dmalan) [inline check50-wrapper]: tbh shouldn't `style50` just ignore all dotfiles?

Comment (kzidane) [reply check50-wrapper]: Sure. Do we wanna remove `-fsanitize=signed-integer-overflow` from Speller's `Makefile`?

**Comment by kzidane** [comment]
https://github.com/cs50/baseimage/pull/25

**Pull request merged**